Contributing/How-to-Join-keploy.md

```diff
@@ -0,0 +1,5 @@
+## Join keploy
+
+Keploy is increasingly looking for new developers to join us and help improve our processes. We can collaborate on developing new projects, improve existing pipelines or maintaining the existing projects. Please follow these steps to join keploy:
+
+1. You can simply Join Keploy Slack Workspace from here and introduce yourself in genral channel [Slack Workspace](https://keploy.slack.com).
```

Comment:
There are other options as well! 

2. There's a Discord channel. 
3. Github Discussions


If you're looking to contribute to anything specific then please feel free to reach out to any of the collaborators of that project directly on Slack/LinkedIn etc.

Reply:
Done